docs/glossary.md

```diff
@@ -340,6 +348,12 @@ Remote data encding is using your custom Data Converter to decode (and encode) y
 
 _Tags: [term](/tags/term), [explanation](/tags/explanation)_
 
+#### [Requests Per Second (RPS)](/references/dynamic-configuration#service-level-rps-limits)
+
+RPS, or Requests Per Second, is used in the Temporal Service (both in self-hosted Temporal and Temporal Cloud). This is a measure that controls the rate of requests at the service level, such as the Frontend, History, or Matching Service.
```

Comment:
Requests per second (RPS) measures the rate at which Service-level requests such as Frontend, History, or Matching Service requests are processed.
Namespaces use RPS rate limits to ensure that Workflow Executions run smoothly, even during request spikes.